### DISCUSSION TITLE
What is the use of `NetworkChangeList`?

### DISCUSSION THREAD

**Discussion by RivanParmar** [android/nowinandroid#822] [Q&A] [answered]
I'm creating an app that uses Firebase for storing data in the cloud. I'm using the "Now In Android" app as an example to implement the synchronization of data from the Firebase server. However, there is this data class `NetworkChangeList` which is confusing me a bit. I don't understand what this class does. I would like to know what it does, so that I could know if I should implement it in my app or not. Moreover, there is no mention of this class anywhere in the [Architecture Learning Journey](https://github.com/android/nowinandroid/blob/main/docs/ArchitectureLearningJourney.md). So, it would be great if it gets added to the docs or if someone could point out it's function.

**Comment by premjit-develops** [comment, marked as answer]
#886 This Discussion's answer may help you.
